config/development.json

```diff
@@ -144,7 +144,7 @@
 		"signup/design-picker-premium-themes-checkout": true,
 		"signup/design-picker-generated-designs": true,
 		"signup/inline-help": false,
-		"signup/goals-step": false,
+		"signup/goals-step": true,
```

Comment:
This should go away after a rebase as it is enabled in the Test PR (for tests to work).

Reply:
Rebasing it